src/seedu/addressbook/parser/Parser.java

```diff
@@ -96,7 +99,10 @@ public Command parseCommand(String userInput) {
                 
             case EditPhoneCommand.COMMAND_WORD:
                 return prepareEditPhone(arguments);
-
+            
```

Comment:
case should align to switch to conform to java coding standards
`switch (condition) {
case ABC:
    statements;
    // Fallthrough
case DEF:
    statements;
    break;
case XYZ:
    statements;
    break;
default:
    statements;
    break;
}`

https://oss-generic.github.io/process/codingstandards/coding-standards-java.html